index.js

```diff
@@ -1,18 +1,95 @@
 /***** Deliverable 1 *****/
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
-
+// Added defer to the HTML Header
 
 /***** Deliverable 2 *****/
-
+header.style.color = "rgb(0, 72, 255)"
+header.style.fontSize = "5rem"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const name = document.querySelector("#profile h2")
+name.textContent = `${traveler.name}`
+
+const nickname = document.querySelector("#profile em")
+nickname.textContent = `${traveler.nickname}`
```

Comment:
great job accessing the traveler data. you don't need string interpolation here since the data is already a string.